src/stub/parser.rs

```diff
@@ -99,6 +122,10 @@ impl<'a, I: Iterator<Item = &'a str>> Parser<I> {
         let mut iter = token.split(":");
         let identifier = String::from(iter.next().unwrap());
         let var_type = iter.next().expect("Error in stub generator: missing type");
+        // NOTE: the regex is false, the length itself can be a variable, f.e:
```

Comment:
Should probably encapsulate in `StringLength` enum with variable and constant option (since variable names will need to be converted to the right case)